tf_quant_finance/black_scholes/brownian_bridge.py

```diff
@@ -0,0 +1,130 @@
+# Lint as: python3
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow.compat.v2 as tf
+
+
+def brownian_bridge_double(*,
+                           x_start,
+                           x_end,
+                           variance,
+                           upper_barrier,
+                           lower_barrier,
+                           n_cutoff=3,
+                           dtype=None,
+                           name=None):
+  """ Probability of not touching the upper barrier and lower_barrier
+  for a Brownian Bridge starting at `x_start`, ending at `x_end` and with
+  variance `variance`, assuming that `x_start` and `x_end` are within the
+  barriers.
+  This can be used in Monte Carlo pricing for adjusting probability of
+  touching the barriers from discrete case to continuous case.
+  Typically in practise, the tensors `x_start`, `x_end` and `variance` should be
+  bi-dimensional (with time steps and paths being the 2 dimensions).
+
```

Comment:
Could you please add an example? See, e.g., [here](
https://github.com/google/tf-quant-finance/blob/ea804648b9461d32675c63844ccae8c4da9f48e3/tf_quant_finance/black_scholes/vanilla_prices.py#L37)